Maquetado/scss/variables.scss

```diff
@@ -0,0 +1,13 @@
+$nav-height: 90px;
```

Comment:
separate in groups by leaving an empty line, group should be: colors, sizes and fonts. another option is to separate it into 3 different files.